EIP_AUTHORS_MANUAL.md

```diff
@@ -0,0 +1,64 @@
+## Introduction
+
+This document outlines the process of specifying and testing EIPs for the Ethereum execution layer. It is intended for EIP authors, researchers and implementers. An EIP will typically go through the following stages:
+
+1. **Research**: The EIP author concieves and refines an idea for an improvement to Ethereum.
+2. **Plain English Specification**: The EIP author writes a document describing the EIP. This is done in the eips repository. [EIPS](https://github.com/ethereum/EIPs/tree/master/EIPS)
+3. **Executable Specifications**: The EIP is implemented in the `execution-specs`. This is intended to make the EIP executable, identify any immediate and obvious implementation issues with the EIP (For example, the eip may not be compatible with some minor detail of the current Ethereum Virtual Machine).
+4. **Writing Tests**: The EIP author writes test schemes for the EIP in the `execution-spec-tests` repository. Having the reference implementation should help identify the various logical flows to test and thus, feed into more robust testing. Once the test schemes are written, the reference implementation can then be used to fill the tests and generate the test vectors.
+5. **Community Discussion**: The EIP is now ready for discussion with the broader Ethereum community. Although the feedback from the community can be sought at all stages of the EIP lifecycle, we feel that having a reference implementation and tests act as a good bridge between research and client implemtation. It also helps core developers (who have limited time and resources) to understand the EIP better and provide more informed feedback.
+6. **Inclusion in a Fork**: Contigent on the community agreeing to the merits of the EIP, it can be included for implementation in a future fork. The test vectors can then be generated using the reference implementation and made available to the client teams.
+7. **Client Implementation**: The EIP is implemented by the various client teams with the test vectors as the guide.
+8. **Mainnet Deployment**: The EIP is deployed on the Ethereum mainnet.
+
+This document will focus on stages 3 and 4 of the above lifecycle.
+
+
+### Executable Specifications
+
+This repository contains the executable specifications for the Ethereum execution layer.
+
+## Folder Structure
+
+# Forks live on mainnet
+
+The folder `src/ethereum` contains the specifications for the different execution layer forks. Each fork has its own folder. For example, the folder `src/ethereum/frontier` contains the specifications for the Frontier hardfork. The `state_transition` function which is available in the `src/ethereum/<FORK NAME>/fork.py` is the transition function for each fork.
+
+# Forks under development
+
+At the time of writing, the Prague Fork is still under development and the previous fork is Cancun, which is live on Mainnet.
```

Comment:
We will constantly forget to update this. Probably better to omit references to specific forks.

Reply:
I'm not sure this needs to be updated at all. I was intending for this to just be a concrete example of the folder structure for the fork under development. Perhaps the fact that this is just an example is unclear from the text. I have updated the text to make that fact more explicit.